Translation corrections + translation sync (#167)
07th-mod/higurashi-console-arcs: Update/tara_001.txt

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#5ec69a>魅音</color>", NULL, "<color=#5ec69a>Mion</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 3, "ps3/s17/03/110300036", 256, TRUE);
	OutputLine(NULL, "「ダメだよ、油性じゃなきゃ。",

> 			NULL, "\"Ahaha, it just has to be permanent.", Line_WaitForInput);

Reviewer: ```suggestion
			NULL, "\"Ahaha, it has to be permanent.", Line_WaitForInput);
```

	ModPlayVoiceLS(4, 3, "ps3/s17/03/110300037", 256, TRUE);
	OutputLine(NULL, "洗ったら落ちちゃうでしょ～？」",
			NULL, " Water-based marker would come off in the laundry.\"", Line_Normal);
	ClearMessage();

//富竹rvS17/08/110800008.「わわ！　なんだいなんだい？！　それで書くの？！」
	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#945c44>富竹</color>", NULL, "<color=#945c44>Tomitake</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 8, "ps3/s17/08/110800008", 256, TRUE);
	OutputLine(NULL, "「わわ！　なんだいなんだい？！　それで書くの？！」",
			NULL, "\"Whoa, whoa! What is this!? You're going to write with that?!\"", Line_Normal);
	ClearMessage();

//rみんなで富竹さんをバックから羽交い締めにし、そこに魅音がマジックを片手にゆっくりと近付いていく。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "みんなで富竹さんをバックから羽交い締めにし、そこに魅音がマジックを片手にゆっくりと近付いていく。",

> 			NULL, "All of us pinned his arms behind his back and Mion approached him with the marker in one hand.", Line_Normal);

Reviewer: ```suggestion
			NULL, "All of us pinned his arms behind his back and Mion approached him with the marker in hand.", Line_Normal);
```

	if (GetGlobalFlag(GADVMode)) { OutputLine("<color=#f0953d>レナ</color>", NULL, "<color=#f0953d>Rena</color>", NULL, Line_ContinueAfterTyping); }
	ModPlayVoiceLS(4, 2, "ps3/s17/02/110200008", 256, TRUE);
	OutputLine(NULL, "「あははは、次に来るときにも着てきてくれるといいな。…いいな！」",

> 			NULL, "\"Ahahaha, you could wear it when you come back. It would be nice!\"", Line_Normal);

Reviewer: ```suggestion
			NULL, "\"Ahahaha, you could wear it next time you come back. It would be nice!\"", Line_Normal);
```

	ClearMessage();

//r感極まった様子だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "感極まった様子だった。",
			NULL, "He appeared deeply moved.", Line_Normal);
	ClearMessage();

//r恥ずかしさとか、他にもいろんな感情のごちゃ混ぜになった、真っ赤な顔だった。
	if (GetGlobalFlag(GADVMode)) { OutputLineAll("", NULL, Line_ContinueAfterTyping); }
	OutputLine(NULL, "恥ずかしさとか、他にもいろんな感情のごちゃ混ぜになった、真っ赤な顔だった。",

> 			NULL, "Embarrassment mixed with a wealth of other emotions made him turn a bright red.", Line_Normal);

Reviewer: ```suggestion
			NULL, "Embarrassment mixed with a wealth of other emotions made him turn bright red.", Line_Normal);
```
